orga/translations/en.json

```diff
@@ -664,6 +664,17 @@
         "row-title": "Participant"
       }
     },
+    "participants-list": {
+      "latest-participation-information-tooltip": {
+        "aria-label": "Latest participation's information",
+        "campaign-name": "Campaign: {campaignName}",
+        "campaign-ASSESSMENT-type": "Type: Assessment",
+        "campaign-PROFILES_COLLECTION-type": "Type : Collecte de profils",
```

Comment:
French text in english translation file.

Reply:
bien vu !!